pom.xml

```diff
@@ -27,6 +27,15 @@
     <junit.version>4.12</junit.version>
     <truth.version>0.36</truth.version>
     <compile-testing.version>0.12</compile-testing.version>
+    <jimfs.version>1.1</jimfs.version>
+    <mockito.version>2.11.0</mockito.version>
+    <ecj.version>4.6.1</ecj.version>
```

Comment:
Must. Kill. ECJ.

Reply:
That stuff is only used in a couple of `@Ignore`d test classes, I doubt they're still relevant